upsetplot/plotting.py

```diff
@@ -272,6 +274,9 @@ class UpSet:
         Whether to label the intersection size bars with the cardinality
         of the intersection. When a string, this formats the number.
         For example, '%d' is equivalent to True.
+    min_subset_size: int, default=1
```

Comment:
```suggestion
    min_subset_size : int, default=1
```
The default should be 0 or -np.inf, or None, since we allow input to be weighted.